types/karma/index.d.ts

```diff
@@ -14,7 +14,10 @@ import Promise = require('bluebird');
 import https = require('https');
 import { Appender } from 'log4js';
 import { EventEmitter } from 'events';
+import * as karmaConstants from './lib/constants';
```

Comment:
this could still be `constants` if you wanted i think

Reply:
> this could still be `constants` if you wanted i think

Yep, again, you're correct. Amended, It now looks much alike the original part.